doc/Config.md

```diff
@@ -122,74 +121,137 @@ $a->config["section"]["key"] = array(
+],
 </pre></td>
 		</tr>
-
 		<tr>
 			<td><pre>
 $a->config["key"] = "value";
 </pre></td>
 			<td><pre>
-[config]
-key = value
+'config' => [
+	'key' => 'value',
+],
 </pre></td>
 		</tr>
-
 		<tr>
 			<td><pre>
 $a->path = "value";
 </pre></td>
 			<td><pre>
-[system]
-urlpath = value
+'system' => [
+	'urlpath' => 'value',
+],
 </pre></td>
 		</tr>
-
 		<tr>
 			<td><pre>
 $default_timezone = "value";
 </pre></td>
 			<td><pre>
-[system]
-default_timezone = value
+'system' => [
+	'default_timezone' => 'value',
+],
 </pre></td>
 		</tr>
-
 		<tr>
 			<td><pre>
 $pidfile = "value";
 </pre></td>
 			<td><pre>
-[system]
-pidfile = value
+'system' => [
+	'pidfile' => 'value',
+],
 </pre></td>
 		</tr>
-
 		<tr>
 			<td><pre>
 $lang = "value";
 </pre></td>
 			<td><pre>
-[system]
-language = value
+'system' => [
+	'language' => 'value',
+],
 </pre></td>
 		</tr>
+	</tbody>
+</table>
+
+#### Migrating from config/local.ini.php to config/local.config.php
+
+The legacy `config/local.ini.php` configuration file is still supported, but is deprecated and will be removed in a subsequent Friendica release.
+
+The migration is pretty straightforward:
+If you had any addon-specific configuration in your `config/addon.ini.php`, just copy `config/addon-sample.config.php` to `config/addon.config.php` and move your configuration values.
+Afterwards, copy `config/local-sample.config.php` to `config/local.config.php`, move the remaining configuration values to it according to the following conversion chart, then rename your `config/local.ini.php` file to check your node is working as expected before deleting it.
 
+<table class="config">
+	<thead>
+		<tr>
+			<th>config/local.ini.php</th>
+			<th>config/local.config.php</th>
+		</tr>
+	</thead>
+	<tbody>
+		<tr>
+			<td><pre>
+[database]
+hostname = localhost
+username = mysqlusername
+password = mysqlpassword
+database = mysqldatabasename
+charset = utf8mb4
+</pre></td>
+			<td><pre>
+'database' => [
+	'hostname' => 'localhost',
+	'username' => 'mysqlusername',
+	'password' => 'mysqlpassword',
+	'database' => 'database',
+	'charset' => 'utf8mb4',
+],
+</pre></td>
+		</tr>
+		<tr>
+			<td><pre>
+[section]
+key = value
+</pre></td>
+			<td><pre>
+'section' => [
+	'key' => 'value',
+],
+</pre></td>
+		</tr>
+		<tr>
+			<td><pre>
+[section]
+key[] = value1
+key[] = value2
+key[] = value3
+</pre></td>
+			<td><pre>
+'section' => [
+	'key' => ['value1', 'value2', 'value3'],
+],
+</pre></td>
+		</tr>
 	</tbody>
 </table>
 
 
+
 ### Database Settings
 
 The configuration variables database.hostname, database.username, database.password, database.database and database.charset are holding your credentials for the database connection.
 If you need to specify a port to access the database, you can do so by appending ":portnumber" to the database.hostname variable.
 
-    [database]
-    hostname = your.mysqlhost.com:123456
+    'database' => [
+        'hostname' => 'your.mysqlhost.com:123456',
+        ...
```

Comment:
Why have you added the dots in the next line? Before the change there were no such dots, copy and pasting the example will get admins into some trouble figuring out why the node does not work anymore I think.

Reply:
Because there are supposed to be other cconfig keys in the `database` section, but they aren't relevant. Specifically for this example, the section shouldn't be copy-pasted. But you're right, I changed my mind about the `...` mid-work.